### ISSUE TITLE
Fix the parameters of first few layers

### ISSUE THREAD

**Issue by zmonoid** [dmlc/mxnet#3708]
There are good examples on how to finetune from a pretrained model for MXNet:
http://shuokay.com/2016/09/02/transfer-learning/
https://ysfalo.github.io/2016/04/01/mxnet%E4%B9%8Bfine-tune/

However, in these examples, actually the gradient of the first few layers were still computed during the training(just the learning rate is different). But I actually only want to train the last layer, and fix the parameters of the first few convolution layers, which would save a lot of time.

Is there anyway to do it in MXNet?

**Comment by piiswrong** [comment]
search for fixed_param_names in module

**Comment by zmonoid** [comment]
@piiswrong Thanks very much!!! Awaresome!

**Comment by miraclewkf** [comment]
I have not got the search result, what do you mean search in module?
